src/main/java/pvpmode/modules/lotr/internal/server/core/LOTRModServerSideHooks.java

```diff
@@ -0,0 +1,52 @@
+package pvpmode.modules.lotr.internal.server.core;
+
+import net.minecraft.entity.player.EntityPlayer;
+import pvpmode.PvPMode;
+import pvpmode.api.common.EnumPvPMode;
+import pvpmode.api.server.utils.PvPServerUtils;
+import pvpmode.internal.server.ServerProxy;
+import pvpmode.modules.lotr.internal.common.LOTRModCompatibilityModuleLoader;
+import pvpmode.modules.lotr.internal.server.LOTRModServerCompatibilityModule;
+
+public class LOTRModServerSideHooks
+{
+    /*
+     * The current one is always false, so returns that by default, of no conditions
```

Comment:
What does this mean?

Reply:
It does mean that at the position where this hook is called, the LOTR code assumes the map location visibility is false, unless the hook here returns true. I'll rephrase the comment and fix the spelling error so that it's clear what I do mean...

Reply:
```suggestion
     * Defaults to false, unless conditions apply.
```
Or better yet, remove this comment, as it is pretty clear from the code.